spec/rails_helper.rb

```diff
@@ -149,12 +149,3 @@
     with.library :action_controller
   end
 end
-
```

Comment:
I see that there are mocks within fog-aws
![Screen Shot 2021-06-30 at 2 57 55 PM](https://user-images.githubusercontent.com/5750239/124016401-9e1ce800-d9b3-11eb-97d0-86e2af388939.png)
, but does this mean we don't need mocking setup all in all, or is there something different going on?

Reply:
In the initializer, I set the testing environment to use the file storage, rather than fog.